### PR TITLE
Adds explanation for why you have to setup your Xcode project for version and build number handling if the project isn't set up properly yet.

### PR DESCRIPTION
**Pull request by Sam0711er**
While I just peeked over the error output, I didn’t really find the answer to my question: "Why do I need to do this now? Do I really want to set it up like that?"

This small yet still effective change helps in better understanding why you should follow the guide :) :tada: ¯\_(ツ)_/¯

### CODE REVIEW

File: fastlane/lib/fastlane/actions/get_build_number.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def self.run(params)
         end
         return build_number
       rescue => ex
-        UI.error('Make sure to follow the steps to setup your Xcode project: https://developer.apple.com/library/ios/qa/qa1827/_index.html')
+        UI.error('Before being able to increment and read the version number from your Xcode project, you first need to setup your project properly. Please follow https://developer.apple.com/library/content/qa/qa1827/_index.html')
```
Comment (milch) [inline fastlane/lib/fastlane/actions/get_build_number.rb]: Maybe instead of just `Please follow...` - `Please follow the guide at...`

**Comment by milch** [comment]
Thanks for your contribution, and congrats to your first fastlane PR! 🚀 

I just have a small suggestion for the text content. 

Also, here, you dropped this: `\`

**Comment by Sam0711er** [comment]
I wanted to change it in order for people to better understand *why* they should follow the guide without having to open it, not to make the language better. 😃

**Comment by milch** [comment]
> I wanted to change it in order for people to better understand why they should follow the guide without having to open it

Yup, you definitely managed to do that. However, the last sentence sounds a bit incomplete, hence my proposed changes. If you could make that change, we'll merge as soon as CI is done running!

**Comment by milch** [comment]
Thanks! 🎉

**Comment by Sam0711er** [comment]
<3 🎉

**Review by milch — APPROVED** [review]
🐶

**Pull request merged**